google-cloud-spanner/src/main/java/com/google/cloud/spanner/Statement.java

```diff
@@ -50,7 +52,9 @@
  *
  * <p>{@code Statement} instances are immutable.
  */
-public final class Statement {
+public final class Statement implements Serializable {
+  private static final long serialVersionUID = -1967958247625065259L;
+
   private final ImmutableMap<String, Value> parameters;
```

Comment:
Value might contain a struct. And since Struct are not serializable, we cannot have Value as serializable implying Statement cannot be serializable.